pandas/core/reshape/pivot.py

```diff
@@ -443,23 +463,25 @@ def pivot(data: "DataFrame", index=None, columns=None, values=None) -> "DataFram
         indexed = data.set_index(cols, append=append)
     else:
         if index is None:
-            index = [Series(data.index, name=data.index.name)]
+            idx_list = [Series(data.index, name=data.index.name)]
         elif is_list_like(index):
-            index = [data[idx] for idx in index]
+            index = cast(Sequence[Label], index)
```

Comment:
you may also be able to get rid of this cast by using convert_to_list_like?

Reply:
ah, yeah, just realize how i misunderstood this cast!

yes, indeed, could use `convert_to_list_like` instead

thanks!